File: 0034-find-first-and-last-position-of-element-in-sorted-array/0034-find-first-and-last-position-of-element-in-sorted-array.java
class Solution {
    public int[] searchRange(int[] nums, int target) {
        int n=nums.length;
        int a=-1,b=-1;
        for(int i=0;i<n;i++){
            if(nums[i]==target){
                if(a==-1){a=i;
                }
                    b=i;
            }
        }
        return new int[] {a,b};
    }
}

File: 0034-find-first-and-last-position-of-element-in-sorted-array/0034-find-first-and-last-position-of-element-in-sorted-array.py
class Solution:
    def searchRange(self, nums: List[int], target: int) -> List[int]:
        n=len(nums)
        a=-1
        b=-1
        ans=[]
        for i in range(0,n):
            if nums[i]==target:
                if a==-1:
                    a=i
                b=i
        return [a,b]